Firestore/core/test/unit/remote/serializer_test.cc

```diff
@@ -1536,18 +1536,20 @@ TEST_F(SerializerTest, EncodesExpectedCount) {
 
   proto.set_resume_token("\001\002\003");
 
-  google::protobuf::Int32Value* expectedCount = new Int32Value();
-  expectedCount->set_value(2);
-  proto.set_allocated_expected_count(expectedCount);
+  google::protobuf::Int32Value int32_value;
+  google::protobuf::Int32Value* expected_count = int32_value.New();
+  expected_count->set_value(1234);
+  proto.set_allocated_expected_count(expected_count);
 
+  EXPECT_TRUE(proto.has_expected_count());
```

Comment:
Can you elaborate on why you have this check in the test:

```
EXPECT_TRUE(proto.has_expected_count());
```

I ask because the line immediately above it explicitly sets the "expected count", so it seems unnecessary to verify that it is, indeed, set. If you think it's okay to remove this check, please do.

Also on line 1569.

Reply:
ok fair enough.